### PR TITLE
change package.json description to refer to Firefox Accounts, not 'PiCl'

### PR DESCRIPTION
**Pull request by jrgm**
Just happened to notice this outdated line. If folks have a better description, let me know.

### CODE REVIEW

**Comment by pdehaan** [comment]
How... How did we miss _that_?

**Comment by ckarlof** [comment]
> How... How did we miss that?

Insufficient linting.

**Comment by pdehaan** [comment]
@ckarlof i'll work on writing and publishing grunt-picl-lint today!

**Comment by ckarlof** [comment]
[Clearly needed](https://github.com/mozilla/fxa-auth-server/pull/316).

**Comment by pdehaan** [comment]
**TL;DR:** There is another reference in **/docs/api.md:683**:
> _... The link can be clicked from any browser, not just the one being attached to the **PICL** account._

---

Well, actually, the **grunt-copyright** task can probably do it. It basically just greps a file anyways. But that would just be absurd.

Here's my hacky `grep` search...
```sh
$ grep -irn "picl" . --exclude-dir={node_modules,.git,load} --exclude=coverage.html
```

1. **./docs/api.md:683:**
  > The verification code will be a random token, delivered in the fragment portion of a URL sent to the user's email address. The URL will lead to a page that extracts the code from the URL fragment, and performs a POST to `/recovery_email/verify_code`. This endpoint should be CORS-enabled, to allow the linked page to be hosted on a different (static) domain. The link can be clicked from any browser, not just the one being attached to the _**PICL**_ account.

1. **./package.json:4:**
  ```js
"description": "An identity provider for PiCL services",
```

---

And all these just seem to be _"identity.mozilla.com/**picl**/v1/"_ namespaces:

1. **./crypto/hkdf.js:8:**
  ```js
const NAMESPACE = 'identity.mozilla.com/picl/v1/'
```

1. **./test/local/pbkdf2_tests.js:13,29,42:**
  ```js
/* L13 */ var salt = Buffer('identity.mozilla.com/picl/v1/first-PBKDF:andré@example.org')
/* L29 */ var salt = Buffer('identity.mozilla.com/picl/v1/first-PBKDF:' + email)
/* L42 */ var salt = Buffer('identity.mozilla.com/picl/v1/second-PBKDF:andré@example.org')
```

1. **./test/local/scrypt_tests.js:19,34:** 
  ```js
/* L19 */ var salt = Buffer('identity.mozilla.com/picl/v1/scrypt')
/* L34 */ var salt = Buffer('identity.mozilla.com/picl/v1/scrypt')
```

**Comment by pdehaan** [comment]
Actually, it's faster to just use GitHub's search https://github.com/mozilla/fxa-auth-server/search?utf8=%E2%9C%93&q=picl and fiddle the URL for each different fxa-* repo.

**Comment by ckarlof** [comment]
> And all these just seem to be "identity.mozilla.com/picl/v1/" namespaces:

That was @warner's doing. He made some arguments about how it's fun to have historical curiosities that everyone forgets the meaning of, but I think it was just his way of leaving his mark.

**Comment by pdehaan** [comment]
Fixed the [./docs/api.md](https://github.com/mozilla/fxa-auth-server/pull/874/files#diff-0f426376b964e609c5c4618eb8bed3b2R683) PiCL reference in #874

**Comment by warner** [comment]
> That was @warner's doing. He made some arguments about how it's fun to have historical curiosities that everyone forgets the meaning of, but I think it was just his way of leaving his mark. 

Yeah, because the "PiCl" acronym is what I most want to be remembered for :).
